config/plugins/interactive_environments/neo/config/allowed_images.yml.sample

```diff
@@ -0,0 +1,11 @@
+# This file lists acceptable images to allow runing.
+#
+# This allows you, the admin, to create multiple flavours
+# for your users to run. E.g. maybe you need a geosciences flavour,
+# you can create the image based on our default image and add the
+# appropriate `apt-get/pip install` statements.
+---
+-
+    image: thoba/neo4j_galaxy_ie
```

Comment:
Ok, sorry for one more thing. Would it be possible that you could pin a specific tag, like we do in jupyter? 

https://github.com/galaxyproject/galaxy/blob/dev/config/plugins/interactive_environments/jupyter/config/allowed_images.yml.sample#L9

This is so if/when we change the GIE API, we can help all of the GIE building groups migrate to a newer version of the API with a new tagged image, and cause fewer "Which image are you using?" issues for end users.